config/appstructure.go

```diff
@@ -987,3 +982,11 @@ func (report *AppStructureReport) AddWarning(warning AppStructureWarning) {
 
 	report.warnings = append(report.warnings, warning)
 }
+
+func truncateMeta(meta *appstructure.MetaDef, truncateTo int) {
+	if truncateTo < 1 {
```

Comment:
Fair, but isn't there a way to impose rules on input argument in config and set to a default if less than 1?

This is added security of course

Reply:
The default is to not truncate